flocker/control/_persistence.py

```diff
@@ -239,8 +310,8 @@ def to_unserialized_json(obj):
     :param obj: An object that can be passed to ``wire_encode``.
     :return: Python object that can be JSON serialized.
     """
-    # Worst implementation everrrr:
-    return loads(wire_encode(obj))
+    # Best implementation everrrr:
+    return _cached_dfs_serialize(obj)
```

Comment:
clearly, the best implementation ever is:

    to_serialized_json = _cached_dfs_serialize

:)